File: Backend/Domain/Device.cs
namespace Domain;

public class Device
{
    public int Id { get; set; }
    public string DeviceName { get; set; }
    public string SerialNumber { get; set; }
    public string Status { get; set; }
    public User? User { get; set; } 
    public int? UserId { get; set; }
    public DateOnly? DateOfIssue { get; set; }
    public DateOnly? DateOfTurnIn { get; set; }
    public string RequestEnum { get; set; }
}

File: Frontend/src/Models/Interfaces/device.ts
export interface Device{
  id: number;
  
  deviceName: string,
  
  serialNumber: string,
  
  status: string,
  
  user?: string;
  userId?: number;
  
  dateOfIssue?: Date;
  dateOfTurnIn?: Date;
  
  requestValue: string;
  requesterId? : number;
  
  errorSubject?: string;
  errorDescription?: string;
 
}
